src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -9,7 +9,10 @@ import MPulseBeacon from '../containers/MPulseBeacon';
 import { ServiceContext } from '../contexts/ServiceContext';
 
 const PageWrapper = ({ children }) => {
-  const { fonts } = useContext(ServiceContext);
+  const { fonts: fontFunctions } = useContext(ServiceContext);
```

Comment:
ideally this should be `const { getFonts } = useContext(ServiceContext);` it makes more sense when functions are named like this

Reply:
Another solution would have been to call them in the `global-styles` component but this was just a simpler, albeit more strange way of doing it I guess.

Reply:
I guess it's fine like this. We can always revisit another time if we want to